day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,24 @@
+def fizz_buzz (num1, num2)
+  (num1..num2).each do |num|
+    if num % 3 == 0 && num % 5 == 0
+      puts "FizzBuzz"
+    elsif num % 3 == 0
+      puts "Fizz"
+    elsif num % 5 == 0
+      puts "Buzz"
+    else
+      puts num
+    end
+  end
+end
+
```

Comment:
Excellent work seeing the patterns for how user input can be leveraged to make all of your day 7 challenges dynamic.